### PR TITLE
support conv1d convert

### PR DESCRIPTION
**Pull request by Flowingsun007**

### CODE REVIEW

**Comment by BBuf** [comment]
ci就有报错：https://github.com/Oneflow-Inc/oneflow_convert/runs/8288666712?check_suite_focus=true

**Comment by Flowingsun007** [comment]
> ci就有报错：https://github.com/Oneflow-Inc/oneflow_convert/runs/8288666712?check_suite_focus=true

fix了

File: oneflow_onnx/oneflow2onnx/handlers/nn.py [hunk 1]
```diff
@@ -196,21 +196,34 @@ def _AddPadding(ctx, node, kernel_shape, strides, dilations=None, spatial=2, cei
...
         else:
-            n, c, h, w = dims
-    dims = [h, w]
+            if node.is_nhwc():
+                n, h, w, c = dims
+            else:
+                n, c, h, w = dims
+        dims = [h, w]
+    else:
+        print("conv3d not support yet!")
```
Comment (BBuf) [inline oneflow_onnx/oneflow2onnx/handlers/nn.py:226]: raise NotImplementedError("xxx")

Comment (Flowingsun007) [reply oneflow_onnx/oneflow2onnx/handlers/nn.py]: 已修改

**Review by BBuf — APPROVED** [review]

**Pull request merged**